docs/concepts.rst

```diff
@@ -1,119 +1,108 @@
+The root directory of your CumulusCI project. Because each project is linked to a single GitHub repository, you can always be confident of the scope of the actions you're taking. And thanks to the keychain, CumulusCI knows which project you are working on by the current working directory of your shell. 
 
-When you work with CumulusCI in a project, the contents of your ``cumulusci.yml`` are merged with the ``cumulusci.yml`` that comes with CumulusCI and defines out-of-the-box functionality. From time to time we'll refer to that as the standard library. Learn more about customizing CumulusCI automation in TODO: link Customization section.
+[NOTE: Save yourself a headache by making sure you're in the correct repository for your project before running project-specific commands; otherwise, your project will produce an error. (This tends to be one of the biggest troubleshooting issues that developers run into with CumulusCI. Check for this first and you might save yourself an extra trip to this guide.)]
 
-Project Source Code
-^^^^^^^^^^^^^^^^^^^
+In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the [TODO link Get Started section].
 
-The main body of the project's code and metadata lives in the ``force-app`` directory for Salesforce DX-format projects and ``src`` for Metadata API-format projects. For managed package projects, only this metadata is part of the package. Most projects have the vast majority of their components stored here.
+cumulusci.yml
+^^^^^^^^^^^^^
 
-Orgs
-^^^^
+The ``cumulusci.yml`` file defines a project's automation in CumulusCI. It contains all the customizations and configurations that pertain to your project's lifecycle. It can encompass everything from customizing the shapes of scratch orgs to configuring tasks and flows.
 
-The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in TODO: link Scratch Org Management. 
+Learn more about customizing CumulusCI automation in the [TODO: link Customization section].
 
-Datasets
+force-app (or src)
+^^^^^^^^^^^^^^^^^^
+
+The main body of the project's code and metadata lives in the ``force-app`` directory for Salesforce DX-format projects and ``src`` for Metadata API-format projects. For managed package projects, only this metadata is part of the package. Projects have the vast majority of their components stored here.
+
+orgs Directory
+^^^^^^^^^^^^^^
+
+The ``.json`` files found in the ``orgs`` directory define the Salesforce DX org configurations that are available to the project. We cover scratch org management in depth in [TODO: link Scratch Org Management].
```

Comment:
"We cover scratch org management in depth"

Haven't mentioned scratch orgs at this point, and there's not enough detail here (yet) that discusses the relationship between orgs directory and scratch orgs. Should we elaborate?

Reply:
I wonder if we should move the info on project structure below the Project Orgs and Services and Tasks and Flows sections, since the project structure includes references to concepts introduced in those sections.

Reply:
Let's definitely set a time to talk about this next week. There's enough of a mix of language in both sections that we might need to sort out what takes priority in getting mentioned first.